gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Sun May 02 10:29:39 MSK 2021
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
```

Comment:
Had to upgrade Gradle (min required version is 7.2.0)
Now it is probably required to use JDK 11 to _build_ the library (but I'm not sure). Shouldn't affect anything else, only the compilation.